easybuild/easyconfigs/p/Perl/Perl-5.26.0-foss-2017b.eb

```diff
@@ -1149,6 +1149,16 @@ exts_list = [
         'source_urls': ['http://search.cpan.org/CPAN/authors/id/D/DE/DEWEG/'],
         'checksums': ['c3d28ad479b14b803bea3ccb8e90cb4c6a2565ee70866a0e9cfe4d76d4d45352'],
     }),
+    ('Math::CDF', '0.1', {
```

Comment:
@smoors Please make this change consistent, add this module in all Perl 5.26.0 easyconfig + more recent ones.